src/sagas/rootSagas.js

```diff
@@ -1,11 +1,20 @@
 import { all } from 'redux-saga/effects';
-
 import headerSaga from 'sagas/timerSaga';
+import driveRulesSaga from 'sagas/rulesSaga';
+import formSaga from 'sagas/formSaga';
 import driveSagas from 'sagas/userDriveSagas';
+import languageSagas from 'sagas/languageSagas';
 import statementRequestSaga from 'sagas/problemsSaga';
-import userSaga from 'sagas/rulesSaga';
-import formSaga from 'sagas/candidateFormSaga';
+import driveTimerSaga from 'sagas/driveTimerSaga';
 
-export default function* rootSaga() {
-  yield all([headerSaga(), driveSagas(), userSaga(), formSaga(), statementRequestSaga()]);
+export default function* rootSagas() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export default function * rootSagas() {
```